x-pack/plugins/monitoring/public/directives/alerts/index.js

```diff
@@ -96,14 +96,19 @@ uiModule.directive('monitoringClusterAlertsListing', kbnUrl => {
 
       scope.$watch('alerts', (alerts = []) => {
         const alertsTable = (
-          <MonitoringTable
-            className="alertsTable"
-            rows={alerts}
-            placeholder="Filter Alerts..."
-            filterFields={filterFields}
-            columns={columns}
-            rowComponent={alertRowFactory(scope, kbnUrl)}
-          />
+          <Fragment>
+            <p className="kuiText kuiVerticalRhythm" tabIndex="0">Showing {alerts.length} alert(s)</p>
```

Comment:
@tsullivan What are your thoughts here?

Reply:
I created #20672 to describe how we might fix this within EUI but I'm still not sure the best place for it here

Reply:
Just to be clear, `MonitoringTable` is our wrapper of `KuiTable` (which hopefully becomes `EuiTable` soon) and streamlines all of our tables. We could add our own count logic there, but it does make sense to have counts appear near the paging controls, which I assume for accessibility should _always_ be visible even if they're disabled.

Reply:
@pickypg I moved the results label to the bottom of the table so it looks like:
<img width="1823" alt="screen shot 2018-07-11 at 2 11 30 pm" src="https://user-images.githubusercontent.com/56682/42591339-6d2473b8-8514-11e8-82d7-740a4e3479f3.png">

I'm not sure that works great from an accessibility perspective as the user will need to tab through all the rows before seeing the total count, but I'm not an expert here to know for sure.

Any thoughts @elastic/kibana-design?

Reply:
We could probably accomplish something like @cchaos is showing by adding it to the right of the filter text box? That seems like a good place for the screen reader to get the data as well, especially since we may not know the _total_ versus what we're displaying.

Reply:
How about:

<img width="1824" alt="screen shot 2018-07-11 at 3 00 32 pm" src="https://user-images.githubusercontent.com/56682/42593709-394e6a4c-851b-11e8-9ecd-00edd5a9bc7d.png">

Reply:
@pickypg @cchaos ^^

Reply:
👍

Reply:
@chrisronline I like it, but I cannot remember: is that count the actual total or just the total displayed on that page? If it's the former, then that seems like the other example. If it's the amount displayed on the current page, then the phrasing is misleading.